### PR TITLE
Add support for browser push

### PR DESCRIPTION
**Pull request by pmngo**
Things to consider before you submit the PR:

* [X] Are tests passing locally?
* [X] Are the files formatted correctly?
* [X] Did you add unit tests?
* [X] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description
This PR adds support for browser push registrations, installations, sends, and hub credentials

### CODE REVIEW

File: src/Microsoft.Azure.NotificationHubs/SRClient.Designer.cs [hunk 1]
```diff
@@ -1211,18 +1247,7 @@ internal class SRClient {
                 return ResourceManager.GetString("FCMRegistrationInvalidId", resourceCulture);
             }
         }
-
```
Comment (pmngo) [inline src/Microsoft.Azure.NotificationHubs/SRClient.Designer.cs:1214]: These values got nuked when I updated the resx file (they were not added there and this file is auto-generated). Will be fixed in a separate PR

**Review by jessHuh — DISMISSED** [review]
lgtm

File: src/Microsoft.Azure.NotificationHubs/SRClient.Designer.cs (hunk 1 above)
Comment (pmngo) [reply src/Microsoft.Azure.NotificationHubs/SRClient.Designer.cs]: Fixed in #309

**Review by jessHuh — APPROVED** [review]

**Comment by islamkhattab** [comment]
Is this pushed to Nuget feed as we do not see it when installing from Nuget Feed, we are using web push but the , also I see the new registration is not added `src/Microsoft.Azure.NotificationHubs/RegistrationDescription.cs`
